### DISCUSSION TITLE
노마드 코더에 실용주의 프로그래머와 클린코드 북클럽 챌린지 라는게 있네요~

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#214] [General]
https://nomadcoders.co/challenges

TIL 형태로 3주동안 책 읽는 과정인데, 저희가 읽은 것을 바탕으로 참가해봐도 좋을듯 합니다~
(3주만에.. 다 읽어야 하는게 좀 빡세긴 하지만... 좀 더 다양한 사람들과 의견을 나눠본다는 관점에서? 해봄직 하네요 ㅎㅎ)

**Comment by jongfeel** [comment]
3주는 너무 빡센데요?? ㅋㅋㅋ

2주 마다 챕터 1 혹은 1~2 읽는 것도 힘들어 하시는 분도 있는데 으흠

**Comment by jongfeel** [comment]
일단 어떤 식으로 진행하고 얼마나 많은 사람들이 탈주(?) 안하고 하는지 알아보기 위해
6월 2일 시작하는 클린코드 북클럽을 신청해 두었습니다 ㅋㅋㅋ

**Comment by TaeHyoungKwon** [comment]
노마드코더에서하는 실용주의 프로그래머 회사분 몇분이랑 같이 참가해봤는데요 체계적으로 잘 만들어두긴 한 것 같습니다 ㅎㅎ 매일마다 해야할 것을 메일로 보내준다던지, 자기랑 다른 사람들 과제 제출여부 상태를 볼 수 있는 페이지를 제공한다던지, 슬랙채널 하나 파놓고 참가하는 사람들끼리 실시간으로 얘기하면서 문의할수있는 것 등등
(다만 하루에 1 장씩 읽어야하는게 압박이라.. 자세히 읽어볼 수 없는 단점이 있습니다. 그래서 가볍게 훑어보는 용도로 본다면 동기부여차원에서 참가해볼만 할듯합니다 ㅋㅋ)

더불어서 노마드코더를 오랜만에 들어가봤는데, 커뮤니티 같은 것도 되게 잘되어있고, 생각보다 활성화 되어있네여 좀 놀랬습니다 ㄷㄷ

---> https://nomadcoders.co/community

**Comment by TaeHyoungKwon** [comment]
## 메일
![image](https://user-images.githubusercontent.com/18538418/168438858-1563b0a0-fb67-409f-8ed6-3182fc40655f.png)


## 과제
![image](https://user-images.githubusercontent.com/18538418/168438850-1dfcfa3b-963a-4a2d-a62f-c2caf0519952.png)


## 상태 페이지
![image](https://user-images.githubusercontent.com/18538418/168438918-5f3e64cb-06c6-44df-afa1-bd7e0a55918e.png)

**Reply by jongfeel** [reply]
오 미리보기급의 설명 감사합니다.

누가 진행 했는지 안했는지 나오는 건 잘 만든거 같네요
